### PR TITLE
Update the runner to use current js(1)

### PR DESCRIPTION
**Pull request by nooga**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -195,7 +195,8 @@ def main() -> None:
         for i, test_file in enumerate(test_files):
             if args.verbose:
                 print(f"Running test: {test_file}")
-            test_result, output = run_test(js, test262, test_file, timeout=args.timeout)
+            test_result, output = run_test(
+                js, test262, test_file, timeout=args.timeout)
```
Comment (linusg) [inline main.py:203]: Please keep the file formatted with [black](https://github.com/psf/black) :)

File: main.py [hunk 2]
```diff
@@ -117,6 +118,9 @@ def success_if(condition: Any) -> Tuple[TestResult, str]:
         return test_result(TestResult.NONZERO_EXIT_ERROR)
     except subprocess.TimeoutExpired:
         return test_result(TestResult.TIMEOUT_ERROR)
+    except:
```
Comment (linusg) [inline main.py:121]: would be nice to capture the exception traceback as output for verbose mode. Something like this I suppose?

```suggestion
    except:
        output = traceback.format_exc()
```

(needs `import traceback` at the top, of course)

**Review by linusg — CHANGES_REQUESTED** [review]

File: main.py [hunk 3]
```diff
@@ -205,6 +210,9 @@ def main() -> None:
             elif test_result == TestResult.NONZERO_EXIT_ERROR:
                 failed_tests += 1
                 emoji = "💥"
+            elif test_result == TestResult.RUNNER_EXCEPTION:
+                failed_tests += 1
+                emoji = "❗️"
```
Comment (linusg) [inline main.py:215]: Oh and how about we swap the emojis with `NONZERO_EXIT_ERROR` here - explosion seems quite fitting for this :D

**Comment by nooga** [comment]
👍 
I'll apply your suggestions in the morning.

**Comment by nooga** [comment]
@linusg there you go :)

File: main.py [hunk 4]
```diff
@@ -2,12 +2,14 @@
...
 import shlex
 import subprocess
+import traceback
 from argparse import ArgumentParser
 from enum import Enum, auto
 from pathlib import Path
 from tempfile import NamedTemporaryFile
 from typing import Any, Iterable, Optional, Tuple
 
+
```
Comment (linusg) [inline main.py:12]:
```suggestion
```

no need for that :)

**Review by linusg — COMMENTED** [review]
I'd prefer to not squash these commits, could you please change the first one to not include the formatting change instead of fixing it in the second? Thanks!

**Comment by nooga** [comment]
> I'd prefer to not squash these commits, could you please change the first one to not include the formatting change instead of fixing it in the second? Thanks!

Erm... I just squashed the whitespace fix before reading this :D

**Comment by linusg** [comment]
Awesome, thank you!!

**Pull request merged**